lib/Service/SlideDeckService.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+namespace OCA\Richdocuments\Service;
+
+use OCA\Richdocuments\AppInfo\Application;
+use OCP\TaskProcessing\Exception\Exception;
+use OCP\TaskProcessing\Exception\NotFoundException;
+use OCP\TaskProcessing\Exception\PreConditionNotMetException;
+use OCP\TaskProcessing\Exception\UnauthorizedException;
+use OCP\TaskProcessing\Exception\ValidationException;
+use OCP\TaskProcessing\IManager;
+use OCP\TaskProcessing\Task;
+use OCP\TaskProcessing\TaskTypes\TextToText;
+use RuntimeException;
+
+class SlideDeckService {
+	public const PROMPT = <<<EOF
+Draft a presentation slide deck with headlines and a maximum of 5 bullet points per headline. Use the following JSON structure for your whole output and output only the JSON array, no introductory text:
```

Comment:
I just remembered, Daphne raised to me that this likely should support different languages as well (especially German). Is that feasible? Just using different prompts then?

Reply:
We should probably add an instruction sentence to make sure the model outputs in the same language as the "presentation text" coming from the user.

The "system" prompt can stay in english. We do it for summary, headlines, topics etc... and it works pretty well with most tested models.